src/mmw/apps/modeling/views.py

```diff
@@ -164,10 +164,15 @@ def start_rwd(request, format=None):
     user = request.user if request.user.is_authenticated() else None
     created = now()
     location = request.POST['location']
+
+    # Parse out the JS style T/F to a boolean
+    snappingParam = request.POST['snappingOn']
```

Comment:
Consider using `request.POST.get('snappingOn', False)` to prevent the possibility of a `KeyError`.